src/main/java/picard/sam/SamToFastq.java

```diff
@@ -414,50 +435,46 @@ private void assertPairedMates(final SAMRecord record1, final SAMRecord record2)
      * messages to be written to the appropriate place.
      */
     protected String[] customCommandLineValidation() {
+
+        List<String> errors = new ArrayList<>();
+
         if (INTERLEAVE && SECOND_END_FASTQ != null) {
-            return new String[]{
-                    "Cannot set INTERLEAVE to true and pass in a SECOND_END_FASTQ"
-            };
+            errors.add("Cannot set INTERLEAVE to true and pass in a SECOND_END_FASTQ");
         }
 
         if (UNPAIRED_FASTQ != null && SECOND_END_FASTQ == null) {
-            return new String[]{
-                    "UNPAIRED_FASTQ may only be set when also emitting read1 and read2 fastqs (so SECOND_END_FASTQ must also be set)."
-            };
+            errors.add("UNPAIRED_FASTQ may only be set when also emitting read1 and read2 fastqs (so SECOND_END_FASTQ must also be set).");

```

Comment:
Can this assertion be done using the `Argument` annotation?

Reply:
As far as i know you can only designate things as mutually exclusive but not mutually inclusive.